shell/browser/serial/serial_chooser_controller.h

```diff
@@ -55,12 +57,21 @@ class SerialChooserController final
   void OnPermissionRevoked(const url::Origin& origin) override {}
   void OnSerialChooserContextShutdown() override;
 
+  // BluetoothAdapter::Observer
+  void AdapterPoweredChanged(device::BluetoothAdapter* adapter,
+                             bool powered) override;
+
  private:
   api::Session* GetSession();
+  void GetDevices();
   void OnGetDevices(std::vector<device::mojom::SerialPortInfoPtr> ports);
   bool DisplayDevice(const device::mojom::SerialPortInfo& port) const;
   void RunCallback(device::mojom::SerialPortInfoPtr port);
   void OnDeviceChosen(const std::string& port_id);
+  void OnGetAdapter(base::OnceClosure callback,
+                    scoped_refptr<device::BluetoothAdapter> adapter);
+  // Whether it will only show ports from bluetooth devices.
+  bool IsWirelessSerialPortOnly();
```

Comment:
_(nit)_ this method can be `const`

```suggestion
  [[nodiscard]] bool IsWirelessSerialPortOnly() const;
```